docs/changelog/104481.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 104481
+summary: Upgrade bundled JDK to 21.0.2
```

Comment:
For some reason this is not present in the release notes (https://www.elastic.co/guide/en/elasticsearch/reference/8.12/release-notes-8.12.1.html).